src/cloudlabs/templates/host_info.html

```diff
@@ -19,9 +19,9 @@ <h1>Information about host '{{ host.label }}'</h1>
     <dt>Cloud provider</dt>
     <dd>Microsoft Azure</dd>
     <dt>VM size</dt>
-    <dd>{{ host.vm_size }}</dd>
+    <dd>{{ host.vm_type }}</dd>
     <dt>Operating System</dt>
-    <dd>{{ host.os_info }}</dd>
+    <dd>Unknown; TODO</dd>
```

Comment:
Is there a reason why we can't show the OS now? Maybe something like
```
"{} {} ()".format(host.os_offer, host.os_sku, host.os_version) if host.os_offer else "Uknown"
```
Or maybe have this as a property on `Host`? (Actually I now see there's a property `os_info` for that, but it seems to work off the Terraform info)

Reply:
Updated!